src/utils/time-range-to-nrql/index.test.js

```diff
@@ -0,0 +1,59 @@
+import { timeRangeToNrql } from '.';
+import {
+  DAY,
+  HOUR,
+  SINCE,
+  UNTIL,
+  DEFAULT_30_MINS_AGO,
+  MINUTES_AGO,
+  HOURS_AGO,
+  DAYS_AGO,
+} from './constants';
+
+test('missing timerange returns 30 mins', () => {
+  expect(timeRangeToNrql({})).toBe(DEFAULT_30_MINS_AGO);
+});
+
+test('empty timerange returns 30 mins', () => {
+  expect(timeRangeToNrql({ timeRange: {} })).toBe(DEFAULT_30_MINS_AGO);
+});
+
+test('conversion of beginTime and endTime', () => {
+  const beginTime = '2 days ago';
+  const endTime = '1 day ago';
+  expect(timeRangeToNrql({ timeRange: { beginTime, endTime } })).toBe(
+    `${SINCE} ${beginTime} ${UNTIL} ${endTime}`
+  );
+});
+
+test('conversion of begin_time and end_time', () => {
+  const begin_time = '2 days ago';
+  const end_time = '1 day ago';
+  expect(timeRangeToNrql({ timeRange: { begin_time, end_time } })).toBe(
+    `${SINCE} ${begin_time} ${UNTIL} ${end_time}`
+  );
+});
+
+test('conversion of duration to minutes', () => {
+  const duration = 3540000;
+  const convertedDuration = 59;
```

Comment:
Or better yet...
```javascript
const durationInMinutes = duration / MINUTE;
```

Reply:
To avoid basing expectations on calculated results that are duplicative of the code being tested: all seed values and expected values are explicitly set, and naming updated to more clearly show what the outcomes the tests are addressing.